tensorflow/python/training/monitored_session.py

```diff
@@ -339,6 +345,14 @@ def MonitoredTrainingSession(master='',  # pylint: disable=invalid-name
   elif save_summaries_steps == USE_DEFAULT:
     save_summaries_steps = None
 
+  if save_checkpoint_steps == USE_DEFAULT and save_checkpoint_secs == USE_DEFAULT:
+    save_checkpoint_steps = None
+    save_checkpoint_secs = 600
+  elif save_checkpoint_secs == USE_DEFAULT:
+    save_checkpoint_secs = None
+  elif save_summaries_steps == USE_DEFAULT:
```

Comment:
not summaries.
could you please add test cases which catch each possible flow.

Reply:
Good catch! 

I have written test case `test_summaries_secs` based on `test_summaries_steps`. what logic condition are you thinking of?